library/L1_Peripheral/lpc40xx/capture.hpp

```diff
@@ -0,0 +1,195 @@
+#pragma once
+#include <cstdint>
```

Comment:
Add newline between `pragma` `c++ libraries` and `sjsu-dev2` libraries, throughout all of the code.